src-tauri/src/main.rs

```diff
@@ -16,13 +16,14 @@ fn interact_with_ai(selection: &str) -> String {
 }
 
 use tauri::Manager;
+use std::env;
 
 fn main() {
     tauri::Builder::default()
         .setup(|app| {
             #[cfg(debug_assertions)] // only include this code on debug builds
             {
-                let dev_tools_visible = false;
+                let dev_tools_visible = env::var("DEV_TOOLS").is_ok();
```

Comment:
Where is this variable defined? Is it in a .env file? And if so, could we add a .env.template file so anyone cloning the repo knows what are the environment variables we are using?

Reply:
Yes, that's an env file. We can also just add a new script to package json like `yarn tauri:dev:tools` or `yarn tauri:dev:debug` what do you think?